keyboards/laptreus/keymaps/debug/rules.mk

```diff
@@ -0,0 +1,3 @@
+ifndef QUANTUM_DIR
```

Comment:
Could you remove this (ifndef/include/endif) block? 

This entire file can be removed, actually.